app/services/arcgis_api/geocoder.rb

```diff
@@ -48,6 +74,34 @@ def request_headers
     end
 
     def parse_suggestions(response_body)
+      handle_api_errors(response_body)
+
+      response_body['suggestions'].map do |suggestion|
+        Suggestion.new(
+          text: suggestion['text'],
+          magic_key: suggestion['magicKey'],
+        )
+      end
+    end
+
+    def parse_address_candidates(response_body)
+      handle_api_errors(response_body)
+
+      response_body['candidates'].map do |candidate|
+        AddressCandidate.new(
+          address: candidate['address'],
```

Comment:
We could always add it in the future if it turns out that we need it. I'm typically in favor of [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it); granted, this whole thing is technically YAGNI since the UI doesn't exist yet 😄